config/routes.rb

```diff
@@ -368,6 +368,8 @@
   get "/tagcollections", to: "tagcollections#index"
   get "/tagcollections/:id", to: "tagcollections#show"
 
+  get "/tagcollections/new", to: "tagcollections#create"
```

Comment:
This was modified on the latest pull down from master. I don't think this PR reflects the most recent changes you have,